### DISCUSSION TITLE
Building plantuml in 2023

### DISCUSSION THREAD

**Discussion by blinxen** [plantuml/plantuml#1540] [Q&A] [answered]
Am I correct to assume that `gradle` is the only supported build tool for plantuml?

The reason I am asking this is because I recently took over the maintenance of plantuml in Fedora. The packaged version for fedora is `1.2023.7`. Currently the sources for plantuml are taken from https://sourceforge.net/projects/plantuml/. The latest version there (`1.2023.7`) is not in sync with GitHub. Also, the sources on sourceforge always included a `build.xml` in the `tar` archive. Is that gone now?

**Comment by The-Lum** [comment, marked as answer]
_[Just for traceability,]_ see answer here: 
- https://github.com/plantuml/plantuml/issues/1542#issuecomment-1726430602

And similar/pseudo request _(about build)_ here:
- #1488

_Thanks to PlantUML team for the enhancement of the `readme`._ 👍 
Regards,
Th.
